src/improvements/tasks/example/README.md

```diff
@@ -0,0 +1 @@
+This `example` directory contains example tasks that are used to make sure templates can always be simulated. When adding a new template, you should add a new task in this directory and make sure that it is simulated via the `template_regression_tests` job in the `circleci/config.yml` file.
```

Comment:
> make sure that it is simulated via the `template_regression_tests` job in the `circleci/config.yml` file.

How do you do this? It looks like that job calls `just simulate-all-templates`, so does this actually just happen automatically?

Reply:
Yes, it'll happen automatically once you add a new task into this directory.

Reply:
got it, so my nit is let's clarify that in the readme, e.g. this:
```suggestion
This `example` directory contains example tasks that are used to make sure templates can always be simulated. When adding a new template, you MUST add a new task in this directory. This will make sure that it is simulated via the `template_regression_tests` job in the `circleci/config.yml` file.
```